src/plugins/github/api.test.js

```diff
@@ -11,18 +11,30 @@ import {
   PULL_REQUEST_REVIEW_NODE_TYPE,
   PULL_REQUEST_REVIEW_COMMENT_NODE_TYPE,
 } from "./types";
+
 describe("GitHub porcelain API", () => {
   const graph = parse(exampleRepoData);
-  // TODO: Create a higher level API that contains all the repositories
-  const repoNode = graph.nodes({type: "REPOSITORY"})[0];
-  const repo = new Repository(graph, repoNode.address);
+  const porcelain = new Porcelain(graph);
+  const repo = porcelain.repository("sourcecred", "example-github");
   function issueOrPRByNumber(n: number): Issue | PullRequest {
     const result = repo.issueOrPRByNumber(n);
     if (result == null) {
       throw new Error(`Expected Issue/PR ${n} to exist`);
     }
     return result;
   }
+  describe("has repository finding", () => {
+    it("which works for an existing repository", () => {
+      expect(
+        porcelain.repository("sourcecred", "example-github")
+      ).toBeDefined();
```

Comment:
This might look inadequate as a test case, but remember that the actual properties on this object are tested under the Wrappers header

Reply:
I have no problem with the breadth of the test case. Note, however, that `expect(null).toBeDefined()` will pass; you might have meant `expect(…).toEqual(expect.anything())`, which passes for everything except `null` and `undefined`. (Not that it matters in this case, but in general this keeps better in line with our philosophy of "try to treat `null` and `undefined` equivalently whenever possible.)